x/auth/vesting/cmd/vestcalc/vestcalc.go

```diff
@@ -431,10 +453,10 @@ var (
 	flagWrite  = flag.Bool("write", false, "Write periods file to stdout.")
 )
 
-// readCmd reads periods in JSON from stdin and writes a sequence of vesting
-// events in local time to stdout.
+// readCmd reads a schedule file from stdin and writes a sequence of vesting
+// events in local time to stdout. See README.md for the format.
```

Comment:
In _local_ time? Oof. If that's the case we should probably find a way to convey UTC offsets.

Reply:
I think it matches the anticipated usage pattern. The comment about timezones in `README.md` describes how to get other timezones.